lib/committee/schema_validator/open_api_3.rb

```diff
@@ -31,10 +31,12 @@ def response_validate(status, headers, response, test_method = false)
           full_body << chunk
         end
 
-        data = {}
-        unless full_body.empty?
-          parse_to_json = !validator_option.parse_response_by_content_type || headers.fetch('Content-Type', nil)&.start_with?('application/json')
-          data = JSON.parse(full_body) if parse_to_json
+        parse_to_json = !validator_option.parse_response_by_content_type || 
+                        headers.fetch('Content-Type', nil)&.start_with?('application/json')
+        data = if parse_to_json
+          full_body.empty? ? {} : JSON.parse(full_body)
+        else
+          full_body
         end
```

Comment:
just thinking if this is readable enough and succinct
```suggestion
        jsonable = !validator_option.parse_response_by_content_type || 
                        headers.fetch('Content-Type', nil)&.start_with?('application/json')
        data = jsonable ? (full_body.empty? ?  {} : JSON.parse(full_body)) : full_body
```

Reply:
It looks a little bit confusing to use two ternary operators in the same line though...